sites.json

```diff
@@ -3694,6 +3708,18 @@
     },
 
     {
+        "name": "Mobills",
+        "url": "https://web.mobillsapp.com/Configuracoes/DeletaConta",
+        "difficulty": "easy",
+        "notes": "If you made an accout via Google or Facebook you must first change your password.",
```

Comment:
I'm not sure I get it. First you have to change the password, then it is just a matter of clicking a delete button? 
If that's so, it is not as direct as just clicking, so I would probably put it as medium

Reply:
The thing is, if you made the account directly on Mobills using an email you can directly delete the account with the button. But if you made the account via Facebook or Google you must take an additional step, so I really don't know how to classify the difficulty

Reply:
I'd probably keep it as easy and rephrase it to something like:
"Just click the delete button. Note that if you have made an account via Google or Facebook, you'll have to change the password first".